tests/Unit/Controller/MessagesControllerTest.php

```diff
@@ -567,11 +662,11 @@ public function testDestroyWithAccountNotFound() {
 		$this->accountService->expects($this->once())
 			->method('find')
 			->with($this->equalTo($this->userId), $this->equalTo($accountId))
-			->will($this->throwException(new DoesNotExistException('')));
+			->willThrowException(new DoesNotExistException(''));
```

Comment:
```suggestion
			->willThrowException(new ClientException());
```

`AccountService.find()` throws a ClientException when encountering an error. You may have to import the exception at the beginning of the file too.